languages/elixir/exercises/concept/basics/.docs/introduction.md

```diff
@@ -0,0 +1,43 @@
+Elixir is a dynamically-typed language, meaning that the type of a variable is only checked at run-time. There is also no real concept of variable assignment.  Using the match `=` operator, we can bind a value to a variable name:
```

Comment:
```suggestion
Elixir is a dynamically-typed language, meaning that the type of a variable is only checked at run-time. Variable names can be _bound_ to a value using the match `=` operator:
```

Reply:
I thought I had fixed that, sorry, will do asap

Reply:
this review is based off of an outdated version, it had been fixed